library/psa_crypto.c

```diff
@@ -7108,6 +7115,13 @@ psa_status_t psa_crypto_init(void)
         return PSA_SUCCESS;
     }
 
+    /* Init drivers */
+    status = psa_driver_wrapper_init();
```

Comment:
I think this part should be updated - it probably should have been worded "The _might be_ a problem" because IIRC it's not that we were sure there was a problem, more that there was something to do and we weren't sure it wouldn't cause issues. Now it's done and have not been causing issues.

Reply:
Ah, that's in `strategy.md`, not `md-cipher-dispatch.md`. In that case, if you don't mind I'd rather fix that as part of #7156 because there's potentially a lot more that's out-of-date in that file right now. I've left a note.